python/sdk/prelude_sdk/controllers/iam_controller.py

```diff
@@ -95,6 +95,39 @@ def create_user(self, permission: int, email: str, expires: datetime = None, nam
             return res.json()
         raise Exception(res.text)
 
+    @verify_credentials
+    def reset_user(self, email: str):
+        """ Reset a user inside an account """
+        res = requests.post(
+            url=f'{self.account.hq}/iam/user/reset',
+            json=dict(handle=email),
+            headers=self.account.headers,
+            timeout=10
+        )
+        if res.status_code == 200:
+            return res.json()
+        raise Exception(res.text)
+
+    @verify_credentials
+    def verify_user(self, token: str):
+        """ Verify a user inside an account """
+        res = requests.get(
+            url=f'{self.account.hq}/iam/user',
+            headers=self.account.headers,
+            params=dict(token=token),
+            timeout=10,
+            allow_redirects=False
+        )
+        if res.status_code != 200:
+            raise Exception(res.text)
+
+        res_json = res.json()
+        if 'token' in res_json:
```

Comment:
what's a case that you get a 200 but no token?

Reply:
anytime you dont input a valid verification_token (either it never existed, or you try to re-use a token that you already successfully verified)